vhdbuilder/packer/components.json

```diff
@@ -245,67 +226,50 @@
       "downloadURL": "mcr.microsoft.com/aks/ip-masq-agent-v2:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.1.9",
         "v0.1.10"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.9",
```

Comment:
removing azure disk & file driver 1.26.* is ok since they are only for aks 1.26, pls keep other images in vhd

Reply:
Will bring the 1.28.6 and 1.29.3 back.